app/src/main/java/com/educacionit/biciya/models/response/ResponseDataClass.kt

```diff
@@ -0,0 +1,26 @@
+package com.educacionit.biciya.models.response
+
+import com.google.gson.annotations.SerializedName
+
+data class StationInformationResponse(
+    val last_updated: Long,
+    val ttl: Int,
+    val data: StationData
+)
+
+data class StationData(
+    val stations: List<Station>
+)
+
+data class Station(
```

Comment:
Can we rename this as StationLocationInfo since this is mostly related to the station location?